bin/fix_my_workspace

```diff
@@ -4,8 +4,17 @@ $VERBOSE = nil  # Don't care about world writable dir warnings and the like
 if $_fix_my_workspace_version_check
   $_fix_my_workspace_version = '0.5.0'
 else
-  require 'origen'
-
+  # if we're running this from a development environment (i.e., bin/ inside of origen repo) then
```

Comment:
Okay. I understand. I didn't know there was another version in origen_updater. I was the running the one directly in <code>bin</code> directory of Origen. Why do we have two of these? Is the usage of the Origen core one deprecated and I just didn't know it.

I don't think this PR breaks the script per se, I think its just a bug in the script which makes debugging a bit wonky. I think that since this is a standalone script, requiring 'origen' is actually requiring the system gem, so the changes I made to Origen aren't being registered unless I explicitly require them. But then, only a few files were included (i.e. only the ones I need :p). I should take a better look at this and make it more robust. Whether this needs to be in this PR or a future improvement is up to you. I guess this will go in the origen_updater anyway so wouldn't be for this PR.

Ultimately, this would work itself out if our system gem origen updates to this PR's version. Then it's the same end result.